### PR TITLE
[RHPAM-3528] Increase KieExecutorMDB threads in RHPAM image

### PR DESCRIPTION
**Pull request by desmax74**
See: https://issues.redhat.com/browse/RHPAM-3528
Signed-off-by: desmax74 <mdessi@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[KIECLOUD-XYZ] Subject`, `[RHDM-XYZ] Subject` or `[RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Review by spolti — DISMISSED** [review]

**Review by jakubschwan — COMMENTED** [review]
Changes looks good to me.
@desmax74 can you please extend behave tests and cover there configuration of the max-pool-size as well? Right now there is only covered max session value. It would by good to have there tests covering default configuration, set only session count with applied default max pool size and also test for configured max session and max pool.

File: jboss-kie-kieserver/added/launch/jboss-kie-kieserver-jms.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ source "${JBOSS_HOME}/bin/launch/logging.sh"
 
 function prepareEnv() {
     # please keep these in alphabetical order
-    unset JBOSS_MDB_MAX_SESSIONS
+    unset JBOSS_KIE_EXECUTOR_MDB_MAX_SESSIONS
```
Comment (spolti) [inline jboss-kie-kieserver/added/launch/jboss-kie-kieserver-jms.sh:9]: this one was left behind :)

**Review by spolti — CHANGES_REQUESTED** [review]

File: jboss-kie-kieserver/added/launch/jboss-kie-kieserver-jms.sh (hunk 1 above)
Comment (desmax74) [reply jboss-kie-kieserver/added/launch/jboss-kie-kieserver-jms.sh:9]: Right :)

Comment (desmax74) [reply jboss-kie-kieserver/added/launch/jboss-kie-kieserver-jms.sh]: fixed

File: tests/features/common/kie-kieserver-common.feature [hunk 2]
```diff
@@ -557,12 +557,15 @@ Feature: Kie Server common features
...
-     And file /opt/eap/standalone/deployments/ROOT.war/WEB-INF/ejb-jar.xml should contain <activation-config-property-name>maxSession</activation-config-property-name>
-     And file /opt/eap/standalone/deployments/ROOT.war/WEB-INF/ejb-jar.xml should contain <activation-config-property-value>987654321123456789</activation-config-property-value>
-
+    And file /opt/eap/standalone/deployments/ROOT.war/WEB-INF/ejb-jar.xml should contain <ejb-class>org.kie.server.jms.executor.KieExecutorMDB</ejb-class>
+    And file /opt/eap/standalone/deployments/ROOT.war/WEB-INF/ejb-jar.xml should contain <activation-config-property-name>maxSession</activation-config-property-name>
+    And file /opt/eap/standalone/deployments/ROOT.war/WEB-INF/ejb-jar.xml should contain <activation-config-property-value>987654321123456789</activation-config-property-value>
+    And container log should contain -Djboss.mdb.strict.max.pool.size=40
+    And file /opt/eap/standalone/configuration/standalone-openshift.xml should not contain <strict-max-pool name="mdb-strict-max-pool" derive-size="from-cpu-count"
```
[the remark this answers is not in the document]
Comment (spolti) [reply tests/features/common/kie-kieserver-common.feature:571]: I think what Max did is correct, it should not contain the derive property if the max session is set

**Pull request merged**